app/src/lib/stores/app-store.ts

```diff
@@ -949,6 +945,32 @@ export class AppStore extends TypedBaseStore<IAppState> {
     })
 
     this.emitUpdate()
+
+    const { branchesState, compareState } = this.getRepositoryState(repository)
+
+    if (branchesState.tip.kind !== TipState.Valid) {
+      return
+    }
+
+    const currentBranch = branchesState.tip.branch
```

Comment:
I believe this cannot be null.

https://github.com/desktop/desktop/blob/595b4e7315dc35fd3d8062d0011ae74e6d41a958/app/src/models/tip.ts#L34-L40